implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java

```diff
@@ -678,6 +692,17 @@ private void processJaxRsMethod(OpenAPIImpl openApi, ClassInfo resourceClass, Me
             createResponseFromJaxRsMethod(method, operation);
         }
 
+        //Add api response using list of exceptions in the methods and exception mappers
+        List<Type> methodExceptions = method.exceptions();
+
+        for (Type type : methodExceptions) {
+            DotName exceptionDotName = type.name();
+            if (exceptionAnnotationMap.keySet().contains(exceptionDotName)) {
```

Comment:
@MikeEdgar What do you think about this one. Do we need to check if the method exception is subclasses of the exception declared in the mapper class?
I can also handle that situation by going up the hierarchy of the method exception to find any parent that match the mapper exception.

Reply:
@lamtrhieu - possibly, but you might pull in too many results. For examples, if a method throws ExceptionA (which extends ExceptionB) and there is a mapper for both, it might be incorrect to include the ExceptionB response information on that particular method. That could always be added in later if it proves to be a desirable feature.